library/src/main/java/com/novoda/downloadmanager/DownloadMigratorBuilder.java

```diff
@@ -1,181 +0,0 @@
-package com.novoda.downloadmanager;
-
-import android.app.Notification;
-import android.app.NotificationChannel;
-import android.content.ComponentName;
-import android.content.Context;
-import android.content.Intent;
-import android.content.ServiceConnection;
-import android.content.res.Resources;
-import android.os.Build;
-import android.os.Handler;
-import android.os.IBinder;
-import android.os.Looper;
-import android.support.annotation.DrawableRes;
-import android.support.annotation.RequiresApi;
-import android.support.v4.app.NotificationCompat;
-import android.support.v4.app.NotificationManagerCompat;
-
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-
-public final class DownloadMigratorBuilder {
```

Comment:
Notifications, Threading and Extraction are now the responsibility of the client application.